libdnf/repo/Repo.cpp

```diff
@@ -1821,12 +1822,13 @@ void
 repo_internalize_trigger(Repo * repo)
 {
     if (repo) {
```

Comment:
While rewriting the function, it might be easier and safer to directly return if repo and hrepo are not set:
```
if (!repo) {
    return;
}
auto hrepo = static_cast<HyRepo>(repo->appdata);
if (!hrepo) {
    return;
}
<remaining code>
```

Doesn't block the review, it's up to you if you decide to change it or not.